README.md

```diff
@@ -1,8 +1,12 @@
 # Scripts and useful commands
 
+## New: CQA modularization checks
+
+[cqa-mod-checks.sh](cqa-mod-checks/cqa-mod-checks.sh) generates a report for modularization issues described in the [Modular documentation templates checklist](https://docs.google.com/document/d/13NAUVAby1y1qfT77QFIZrMBhi872e7IEvAC9MUpGXbQ/edit?tab=t.0) and [CQA template](https://docs.google.com/spreadsheets/d/11LyS_q40rF0IQ0p-U-ZG1legKHB7dKbv8Kn279wqvpA/edit?usp=drive_link). It uses the input files generated by [extract-doc-modules.sh](extract-doc-modules.sh).
+
```

Comment:
It would be great to mention which Modularization critera from the CQA these checks help evaluate, e.g.

- Modularization / All Required/non-negotiable modular elements are present
- Modularization / Content does not include disallowed modular elements
- Anything else?